themes/edx.org/lms/templates/header.html

```diff
@@ -74,6 +103,39 @@ <h2 class="course-header">
         % endif
       % endif
 
+    % if uses_pattern_library:
```

Comment:
This *appears* to be identical to the dropdown in the non-edX version. Can we make this into a shared template? It would make it simpler to add new items in the future.